app/controllers/admin/discourse_calendar/admin_holidays_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "holidays"
+
+module Admin::DiscourseCalendar
+  class AdminHolidaysController < AdminDiscourseCalendarController
+    def index
+      begin
+        holidays = Holidays.year_holidays([params[:id]], Time.current.beginning_of_year)
+      rescue Holidays::InvalidRegion
+        raise Discourse::NotFound
+      end
+
+      render json: { holidays: holidays }
```

Comment:
Not necessary in this PR, but we might want to include the region name label here, the UI will likely need it. (Same for the holiday regions controller.)

Reply:
Ah yes, of course! An oversight on my part, I suppose it would have surfaced at some point! :sweat_smile: :smiley: I can take care of it here, something like this for the return payload?:
```
{ region: params[:id], holidays: holidays }
```

Or thoughts on `region_code` or `region_id`? :thinking:

Reply:
I was thinking it would be useful to have both the region code (`ar`) and a pretty name to display (like `Holidays in Argentina`) but it doesn't look like the gem has that. So might be best to leave this for later, we'll probably want to just use localizations based on the region code.

Reply:
> I was thinking it would be useful to have both the region code (ar) and a pretty name to display (like Holidays in Argentina) but it doesn't look like the gem has that. 

Ah, makes sense! Well, to start I updated this to return the region code at least. :smiley: